src/pages/registration/components/RegistrationNotFound.tsx

```diff
@@ -0,0 +1,39 @@
+import { Container, Heading, Flex, Text, Divider } from '@chakra-ui/layout';
+import { Button, Box } from '@chakra-ui/react';
+import { HiExternalLink } from 'react-icons/hi';
+import { useParams } from 'react-router';
+
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+import { getReadableTerm } from 'lib/utils/terms';
+
+export function RegistrationNotFound() {
+  const { term } = useParams();
+  const mode = useDarkMode();
+  return (
+    <Container maxW="container.xl" centerContent data-testid="registration-component-1">
+      <Divider my="4" />
+      <Box padding="10">
+        <Flex direction={{ md: 'row', base: 'column' }} justifyContent="space-between">
+          <Heading size="md" color={mode('gray', 'dark.header')}>
+            Unable to find saved courses from{' '}
+            <Text as="span" color={mode('black', 'white')}>
```

Comment:
```suggestion
            <Text as="span">
```
Might need to test this to double check, but I think chakra automatically handles black and white text so this

Reply:
Nevermind, this is needed because it is nested in text that has a different colour